docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -4785,7 +4785,6 @@ image](#create-an-image) section for more details.
     Content-Type: application/json
 
     {
-      "Name": "top",
```

Comment:
I tried doing an API request that omitted the "Name", and I think it worked, but I'll double check with the current code (as I made some modifications since, and possibly I had `request.Spec.Annotations.Name != nil && ...` in the previous version).